CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing
+
+Community contributions to Datadog projects for Ruby are welcome! See below for some basic guidelines.
+
+## Want to request a new feature?
+
+Many great ideas for new features come from the community, and we'd be happy to consider yours!
+
+To share your request, you can [open a Github issue](https://github.com/DataDog/dd-trace-rb/issues/new) with the details about what you'd like to see. At a minimum, please provide:
```

Comment:
Couple places referring to dd-trace-rb, is that intended?

Reply:
Nope, an oversight